automatic-off-session-payment/templates/booking-request-accepted/booking-request-accepted-html.html

```diff
@@ -39,7 +38,7 @@ <h1>Your booking request was accepted!</h1>
     {{#with transaction}}
     <p>{{provider.display-name}} has accepted your booking request for {{listing.title}} from {{> format-date date=booking.start}} to {{> format-date date=booking.end}}.</p>
 
-    <p>We have charged {{> format-money money=payin-total}} from your credit card. Here's your receipt.</p>
+    <p>We will charge {{> format-money money=payin-total}} from the credit card you have saved in {{marketplace.name}} Payment Methods one day before the booking starts. Here's the breakdown of your upcoming charge.</p>
```

Comment:
I guess payment methods could also be written with lowercase as it's referring to payment methods in general. Or would there be a reason to use upper case here?